knowledge-base/adr/outdated/0001-circuit-breaker-options.md

```diff
@@ -1,6 +1,10 @@
 # Circuit Breaker Options
 
-_Date_: 2019-07-29
+## Status
+
+**superseded** 
+
+We will add resilience via middleware approach ([ADR#0031](../0031-resilience-options.md)).
 
 ## Context: What Triggered this Discussion?
```

Comment:
[q] Do we need to align the superseded/outdated ADRs to also follow the template header titles? For example in this one, we have additional text in Context title and an ##Idea section